### PR TITLE
fix: missing mediatr dependency

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/BuildingRegistry.Api.CrabImport/Infrastructure/Modules/MediatRModule.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public class MediatRModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder
+                .RegisterType<Mediator>()
+                .As<IMediator>()
+                .InstancePerLifetimeScope();
+
+            // request & notification handlers
```
Comment (lvermeulen) [inline src/BuildingRegistry.Api.CrabImport/Infrastructure/Modules/MediatRModule.cs:18]: this block should not be there anymore (MediatR 12.0.1)

Comment (ArneD) [reply src/BuildingRegistry.Api.CrabImport/Infrastructure/Modules/MediatRModule.cs:18]: it's still 10.x

**Review by lvermeulen — COMMENTED** [review]
change requested

**Review by ArneD — APPROVED** [review]

**Pull request merged**